### PR TITLE
BDLaethg43

### PR DESCRIPTION
**Pull request by thea-m**
And another scroll :)

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethg43.xml [hunk 1]
```diff
@@ -50,35 +161,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <date notBefore="1800" notAfter="2006"/>
+                                <desc>Relatively regular hand. Broad and slightly cursive characters. The word separator at the end of a line is 
+                                occasionally omitted or placed at the beginning of the next line. The ligature
+                                <foreign xml:lang="gez"><hi rend="ligature">ግዚ</hi></foreign> is used.</desc>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">Incipits (one or two rubricated lines), name of the owner, one numeral, 
+                                    the words <foreign>ወይርኃቁ፡</foreign>
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethg43.xml:243]: `<foreign xml:lang="gez">ወይርኃቁ፡</foreign>`?

File: OxfordBodleian/Juel-Jensen/BDLaethg43.xml [hunk 2]
```diff
@@ -121,11 +353,112 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                        </div>
+                        
+                        <div type="textpart" subtype="text" n="7" corresp="#ms_i7" xml:id="text7">
+                            <ab>
+                                <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡</hi> ቅዱስ፡ ፩ አምላክ፡ ጸሎት፡ በእንተ፡ ሕማመ፡ ቍርፀት፡ ዘተፈነወ፡ 
+                                እምኀበ፡
+                                <sic>ወወል፡</sic> ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ይርዳዕ፡ ወይዕ፡ ወይቤዘ፡ ለውሉደ፡ እጓለ፡ እምሕያው፡ እንዘ<supplied reason="omitted">፡</supplied>
+                                ይበ<gap reason="illegible" unit="chars" quantity="1"/>ብዕ፡ ከርሥ፡ ወያጠዊ፡ ዓማዑት፡ ወያደቅቅ፡ አዕፅምት፡ ሸገር፡ ሸገር፡ በኃይለ፡ ዝንቱ፡ 
+                                አስማቲከ፡
+                                አድኅኖ፡ እምሕማማ፡ ቍርፀት፡ ለገብርከ። <persName ref="PRS13851TaklaM"><hi rend="rubric">ተክለ፡ ማርያም።</hi></persName>
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethg43.xml:435]: እምሕማማ፡ or እምሕማመ፡?

**Comment by thea-m** [comment]
Thank you Massimo, added everything

**Review by ElianaDalSasso — APPROVED** [review]
Thank you Thea!

**Pull request merged**